core/dbt/contracts/graph/manifest.py

```diff
@@ -985,6 +988,20 @@ def analysis_lookup(self) -> AnalysisLookup:
             self._analysis_lookup = AnalysisLookup(self)
         return self._analysis_lookup
 
+    @property
+    def pydantic_semantic_manifest(self) -> Optional[PydanticSemanticManifest]:
```

Comment:
Borderline nitpicky, the return type is actually a non-Optional PydanticSemanticManifest, which is the more attractive return type anyway.

Reply:
Oh right, good catch! I had planned on having it return `None` if there were not any SemanticModel or Metric nodes 🤦 I think returning an empty semantic model is a pretty good default. I'll get the typing updated, and then also update `write_semantic_manifest` to not have to anticipate it ever being `None`